integration-tests/node-manifest/__tests__/create-node-manifest.test.js

```diff
@@ -1,13 +1,14 @@
+// Must run `gatsby-dev --packages gatsby to get latest gatsby changes before running the tests
 jest.setTimeout(100000)
+const DEFAULT_MAX_DAYS_OLD = 30
 
 const { spawnGatsbyProcess } = require(`../utils/get-gatsby-process`)
 const urling = require(`urling`)
 const rimraf = require(`rimraf`)
 const path = require(`path`)
 const fs = require(`fs-extra`)
-const {
-  getPageDataDigestForPagePath,
-} = require(`gatsby/dist/utils/node-manifest`)
+
+const gatsbyCommandName = process.env.GATSBY_COMMAND_NAME || `develop`
```

Comment:
I know this was in the old code too but what is this?

Reply:
We run the test suite with `develop` and `build`. `"test": "gatsby clean && GATSBY_COMMAND_NAME=build jest && gatsby clean && GATSBY_COMMAND_NAME=develop jest"`

Reply:
This is mainly to support legacy preview. We can remove it soon now that Sanity will be on PINC and all our other major integrations are. I'd like to have a bit of overlap though before we remove it